src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -128,6 +129,36 @@ public void testKeyEncryption() throws InvalidParametersException {
     }
 
     @Test
+    public void testDefaultAesCipherParams() throws Exception {
+        //plaintext is less than 1 block long
+        String plaintextLength28 = StringHelper.generateStringOfLength(7, 'A');
+        assertEncryptionUsesExpectedDefaultParams(plaintextLength28);
+
+        //plaintext is equal to 1 block
+        String plaintextLength128 = StringHelper.generateStringOfLength(32, 'A');
+        assertEncryptionUsesExpectedDefaultParams(plaintextLength128);
+
+        //plaintext is more than 1 block long
+        String plaintextLength136 = plaintextLength128 + StringHelper.generateStringOfLength(2, 'A');
+        assertEncryptionUsesExpectedDefaultParams(plaintextLength136);
+    }
+
+    private static String encryptWithoutSpecifyingAlgorithmParams(String plaintext) throws Exception {
+        SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+        Cipher cipher = Cipher.getInstance("AES");
+        cipher.init(Cipher.ENCRYPT_MODE, sks, cipher.getParameters());
+        byte[] encrypted = cipher.doFinal(plaintext.getBytes());
+        return StringHelper.byteArrayToHexString(encrypted);
+    }
```

Comment:
This method should not be placed above where it is used.  Move it below the method `assertEncryptionUsesExpectedDefaultParams`